### PR TITLE
Streamline usage

### PR DESCRIPTION
**Pull request by hglman**
- Use two files with a small set of columns to build the dataset to train on.
- Refactor the way files are read into data frames allowing a number of file formats.
- Flatten the output so there are not arrays in columns for easier manipulation in say excel.

### CODE REVIEW

File: cv19index/io.py [hunk 1]
```diff
@@ -4,41 +4,98 @@
...
     with open(fpath, "rb") as fobj:
         return pickle.load(fobj)
 
 
-def read_frame(fpath, schema_path=None, empty_ok=False):
-    if schema_path is None:
-        schema_path = fpath + ".schema.json"
+def get_na_values(dtypes):
+    # Pandas converts a string with "NA" as a real NaN/Null. We don't want this
+    # for real string columns. NA can show up as a real flag in customer data and
```
Comment (DaveDeCaprio) [inline cv19index/io.py]: Get rid of "customer data" here since this is an open source project.

File: cv19index/predict.py [hunk 2]
```diff
@@ -354,33 +364,48 @@ def do_run(
...
+        logger.error(f"{args.model} is not a valid model type. valid models are xgboost.")
+
+
+def parser():
     logging.basicConfig(level=logging.INFO)
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("input")
-    parser.add_argument("output")
+    parser.add_argument("demo")
```
Comment (DaveDeCaprio) [inline cv19index/predict.py]: demo is confusing.  Not obvious that it is short for demographics.  Call it person or demographics.

File: cv19index/resources/xgboost/claims.schema.json [hunk 3]
```diff
@@ -0,0 +1,132 @@
+{
+  "schema": [
```
Comment (DaveDeCaprio) [inline cv19index/resources/xgboost/claims.schema.json:2]: Not sure why we have these.  We shouldn't be requiring people to set up a schema.json file for their inputs.  They shoudl be able to just send in a file.

File: examples/data/.~lock.claims.xls# [hunk 4]
```diff
@@ -0,0 +1 @@
+,ben,benlaptop,22.03.2020 18:53,file:///home/ben/.config/libreoffice/4;
```
Comment (DaveDeCaprio) [inline examples/data/.~lock.claims.xls#]: delete this file

File: examples/data/.~lock.outpatient.csv# [hunk 5]
```diff
@@ -0,0 +1 @@
+,ben,benlaptop,22.03.2020 18:41,file:///home/ben/.config/libreoffice/4;
```
Comment (DaveDeCaprio) [inline examples/data/.~lock.outpatient.csv#]: delete this file

**Review by DaveDeCaprio — CHANGES_REQUESTED** [review]
README needs to be updated also.
Not sure what you did with the tutorial notebook and the old sample data.  We should probably remove that since it was confusing and the tutorial notebook shoudl essentially replicate the guts of the main function, showing people the data.

File: cv19index/resources/xgboost/claims.schema.json (hunk 3 above)
Comment (hglman) [reply cv19index/resources/xgboost/claims.schema.json:2]: The IO parser uses them to get the needed column set. I figured they could be used to outline what columns you did need?

File: cv19index/predict.py (hunk 2 above)
Comment (hglman) [reply cv19index/predict.py]: Agreed, has been changed to demographics.

File: cv19index/resources/xgboost/claims.schema.json (hunk 3 above)
Comment (DaveDeCaprio) [reply cv19index/resources/xgboost/claims.schema.json:2]: Ah, so this is just the "needed columns".  They can have whatever other columns they want and it's ok?  That's fine.

Comment (hglman) [reply cv19index/resources/xgboost/claims.schema.json:2]: Yes, that's correct. These are the required columns, extra columns will not fail processing.

File: cv19index/io.py [hunk 6]
```diff
@@ -48,75 +105,27 @@ def adj_type(x):
...
-        or x["dataType"]["dataType"] == "datetime"
-    ]
-    # Pandas read_csv ignores the dtype for the index column,
-    # so we don't read in with an index column.  Set it later.
     df = pd.read_csv(
         fpath,
         header=0,
-        names=names,
-        dtype=adj_dtypes,
+        dtypes=adj_dtypes,
```
Comment (davesullivan41) [inline cv19index/io.py]: This should be dtype, not dtypes

**Review by DaveDeCaprio — APPROVED** [review]

**Pull request merged**